.github/workflows/sycl-windows-run-tests.yml

```diff
@@ -65,6 +65,7 @@ jobs:
     - name: Extract SYCL toolchain
       shell: bash
       run: |
+        rm -rf install
```

Comment:
Please drop these changes from this PR.

Reply:
ok, moved to separate PR: https://github.com/intel/llvm/pull/15532